### PR TITLE
Update debian base image to 1.9.0

### PR DESCRIPTION
**Pull request by saikat-royc**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
removed the symlink creation as it already exists in 1.9.0 (```failed to create symbolic link '/usr/sbin/rm': File exists```)
**Does this PR introduce a user-facing change?**:

```release-note
Update debian base image to 1.9.0
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,16 +24,14 @@ ADD . .
...
+RUN clean-install udev
 
 # Start from Kubernetes Debian base
-FROM k8s.gcr.io/build-image/debian-base:buster-v1.6.0
+FROM k8s.gcr.io/build-image/debian-base:buster-v1.9.0
 COPY --from=builder /go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/bin/gce-pd-csi-driver /gce-pd-csi-driver
 # Install necessary dependencies
-RUN ln -s /bin/rm /usr/sbin/rm \
-  && clean-install util-linux e2fsprogs mount ca-certificates udev xfsprogs
+RUN clean-install util-linux e2fsprogs mount ca-certificates udev xfsprogs
```
Comment (jingxu97) [inline Dockerfile:34]: what is the reason for this change?

Comment (saikat-royc) [reply Dockerfile:34]: When I changed the image to 1.9.0, the build was failing with error `failed to create symbolic link '/usr/sbin/rm': File exists`
